src/components/map/buttons/NavigationButtons.vue

```diff
@@ -28,13 +28,28 @@ const panelStore = usePanelsStore()
 const mapStore = useMapStore()
 const simulationStore = useSimulationStore()
 
+function IsPanelVisibleOnStep() {
```

Comment:
Can you rename the function please , like this: isSolarPanelVisibleOnStep
Panel is also the name for the left panel, it can be confused